bazel/packaging/BUILD

```diff
@@ -17,7 +17,7 @@ redpanda_package(
 # This flag is used to set the interpret path for redpanda binary.
```

Comment:
comment is wrong?

Maybe we should call this `staging_dir`, I think install_path doens't make much sense now?